application/src/main/java/org/thingsboard/server/service/cloud/BaseUplinkMessageService.java

```diff
@@ -62,18 +51,35 @@
 
 @Slf4j
 public abstract class BaseUplinkMessageService {
-
-    protected static final String QUEUE_SEQ_ID_OFFSET_ATTR_KEY = "queueSeqIdOffset";
+    private static final String CANT_PROCESS_UPLINK_RESPONSE_MESSAGE = "Can't process uplink response message";
+    private static final String MSG_HAS_BEEN_PROCESSED_SUCCESSFULLY = "Msg has been processed successfully!";
+    private static final String MSG_PROCESSING_FAILED = "Msg processing failed!";
+    private static final String EVENTS_ARE_GOING_TO_BE_CONVERTED = "event(s) are going to be converted.";
+    private static final String INTERRUPTED_WHILE_WAITING_FOR_LATCH = "Interrupted while waiting for latch. ";
+    private static final String UPLINK_MSGS_ARE_GOING_TO_BE_SEND = "uplink msg(s) are going to be send.";
+    private static final String INTERRUPTED_EXCEPTION = "sendUplinkMsgPack throw InterruptedException";
+    private static final String UPLINK_MSG_SIZE_ERROR_MESSAGE =
+            "Uplink msg size [{}] exceeds server max inbound message size [{}]. Skipping this message. " +
+                    "Please increase value of EDGES_RPC_MAX_INBOUND_MESSAGE_SIZE env variable on the server and restart it. Message {}";
+    private static final String FAILED_TO_DELIVER_THE_BATCH = "Failed to deliver the batch:";
+    private static final String ERROR_DURING_SLEEP = "Error during sleep between batches or on rate limit violation";
+    private static final String NEXT_MESSAGES_ARE_GOING_TO_BE_DISCARDED = "Next messages are going to be discarded";
+    private static final String CONVERTING_CLOUD_EVENT = "Converting cloud event";
+    private static final String EXCEPTION_DURING_CONVERTING_EVENTS = "Exception during converting events from queue, skipping event";
+    private static final String UNSUPPORTED_ACTION_TYPE = "Unsupported action type";
+    private static final String EXECUTING_CONVERT_ENTITY_EVENT = "Executing convertEntityEventToUplink";
+    private static final String UNSUPPORTED_CLOUD_EVENT_TYPE = "Unsupported cloud event type";
```

Comment:
I am not a fan of this constants.. I prefer to see it inside the log, especially since most of them using with 1 time